migration/mule/service-orchestration-and-choice-routing/README.md

```diff
@@ -0,0 +1,143 @@
+    ```sql
+    CREATE USER 'process_manager'@'localhost' IDENTIFIED BY 'process123';
+    GRANT ALL PRIVILEGES ON `OrderProcess`.`orders` TO 'process_manager'@'localhost';
+    GRANT ALL PRIVILEGES ON `OrderProcess`.`order_audits` TO 'process_manager'@'localhost';
+    ```
+7. Start WSO2 Integration Studio ([Installing WSO2 Integration Studio](https://ei.docs.wso2.com/en/latest/micro-integrator/develop/installing-WSO2-Integration-Studio/)).
+8. In your menu in Studio, click the **File** menu. In the File menu select the **Import...** item.
+9. In the Import window select the **Existing WSO2 Projects into workspace** under **WSO2** folder.
+10. Browse and select the file path to the downloaded sample of this Github project 
+(`integration-studio-examples/migration/mule/service-orchestration-and-choice-routing`) and click **finish**.
+11. Run the sample by right click on the **ProcessOrderCompositeApplication** under the main 
+**service-orchestration-and-choice-routing** project and selecting **Export Project Artifacts and Run**.
+12. Open HTTP Client in Integration Studio. Follow [HTTP Client Guidelines](../../../docs/common/adding-http-client-to-integration-studio.md)
+to open HTTP Client if the window is not visible in the interface.
+13. Make a POST request to `http://localhost:8290/services/Orders`.
+14. Add following `SOAPAction` and `Content-Type` headers.
+    - SOAPAction: processOrder
+    - Content-Type: text/xml
+15. Add the following input payload.
+    ```xml
+    <soapenv:Envelope 
+     	xmlns:soapenv="http://schemas.xmlsoap.org/soap/envelope/" 
+     	xmlns:ord="http://orders.wso2.org/">
+        <soapenv:Header/>
+        <soapenv:Body>
+           <ord:processOrder>
+                 <order>
+                 <orderId>1223</orderId>
+                 <customer>
+                    <address>Main Street 123</address>
+                    <firstName>John</firstName>
+                    <lastName>Doe</lastName>
+                 </customer>
+                <item>
+                   <manufacturer>Samsung</manufacturer>
+                   <name>s-10</name>
+                   <productId>SM350</productId>
+                   <quantity>1</quantity>
+                </item>
+                <item>
+                   <manufacturer>SoftSell</manufacturer>
+                   <name>B12</name>
+                   <productId>LK12</productId>
+                   <quantity>2</quantity>
+                </item>              
+              </order>
+           </ord:processOrder>
+        </soapenv:Body>
+     </soapenv:Envelope>
+    ```
+16. Following response can be observed in the HTTP Response pane.
+    ```xml
+    <?xml version='1.0' encoding='UTF-8'?>
+    <soap:Response xmlns:soap="http://schemas.xmlsoap.org/soap/envelope">
+        <message>"Order 1223 was processed successfully" </message>
+    </soap:Response>
+    ```
+	
+### How It Works
```

Comment:
The message flow is not clearly described. Shall we describe the message flow briefly? The reader should grab what services are called in what order.